test/quadratic.jl

```diff
@@ -49,7 +49,7 @@ xlo, xhi = itp3[.2], itp3[xmax+.7]
 @test_approx_eq xhi A[end]
 
 # Check continuity
-xs = [0:.1:length(A)+1]
+xs = [0:.1:length(A)+1;]
```

Comment:
We should actually be able to do without either; we could just as well do

```
for x in 0:.1:xmax
    @test_approx_eq_eps itp3[x] itp3[x+.1] .1
end
```

which I think is much less magical. (But I'm being *really* nitpicky here, so I don't blame you if you ignore me...)